StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/CardSectionWithScanner/CVCRecollectionElement.swift

```diff
@@ -26,18 +30,36 @@ final class CVCRecollectionElement: Element {
     var isViewInitialized: Bool = false
     lazy var cvcRecollectionView: CVCRecollectionView = {
         isViewInitialized = true
-        return CVCRecollectionView(defaultValues: defaultValues,
-                                   paymentMethod: paymentMethod,
-                                   mode: mode,
-                                   appearance: appearance,
-                                   elementDelegate: self)
-
+        return CVCRecollectionView(
+            defaultValues: defaultValues,
+            paymentMethod: paymentMethod,
+            mode: mode,
+            appearance: appearance,
+            textFieldView: textFieldElement.view
+        )
     }()
 
     let defaultValues: DefaultValues
     var paymentMethod: STPPaymentMethod
     let appearance: PaymentSheet.Appearance
 
+    lazy var textFieldElement: TextFieldElement = {
+        let textFieldElement = TextFieldElement(configuration: cvcElementConfiguration, theme: appearance.asElementsTheme)
+        textFieldElement.view.backgroundColor = appearance.colors.componentBackground
```

Comment:
Does passing in the theme/appearance not do this already? Same question with the corner radius.

Reply:
Probably not, this was carried over. I can try to remove.

Reply:
Yep, unnecessary. There's more we could do to clean this element up fwiw but I don't want to do more than necessary right now.